src/lib/fhirTables.js

```diff
@@ -397,3 +413,196 @@ function obsCompare(a, b) {
   return(effectiveB - effectiveA);
 }
 
+// +--------------------+
+// |    Plan of Care    |
+// +--------------------+
+
+function carePlanHeader() {
+  return (
+    <tr>
+      <th>Status</th>
+      <th>Intent</th>
+      <th>Activities</th>
+      <th>Category</th>
+      <th>Period Start</th>
+      {/* Add headers for other relevant CarePlan properties */}
+    </tr>
+  );
+}
+
+function carePlanRow(r, rmap, dcr) {
+  const status = r.status;
+  const intent = r.intent;
+
+  const activities = futil.joinJSXElements(
+    (r.activity || []).map(activity =>
+      activity.detail && activity.detail.code
+      ? futil.renderCodeableJSX(activity.detail.code, dcr)
+      : null
+    ).filter(activity => activity !== null),
+    ', '
+  );
+
+  const category = futil.joinJSXElements(
+    (r.category || []).map(c =>
+      c.text ? futil.renderCodeableJSX(c, dcr) : null
+    ).filter(c => c !== null),
+    ', '
+  );
+
+  const period = r.period ? futil.renderPeriod(r.period) : "";
+
+  return (
+    <tr key={r.id}>
+      <td>{status}</td>
+      <td>{intent}</td>
+      <td>{activities}</td>
+      <td>{category}</td>
+      <td>{period}</td>
+      {/* Render other relevant CarePlan properties as table cells */}
+    </tr>
+  );
+}
+
+// +--------------------+
+// |      Consent       |
+// +--------------------+
+
+function consentHeader() {
+  return (
+    <tr>
+      <th>Status</th>
+      <th>Scope</th>
+      <th>Category</th>
+      <th>Date/Time</th>
+      <th>Policy Rule</th>
+      <th>Provision Period</th>
+      <th>Organization</th>
+    </tr>
+  );
+}
+
+function consentRow(r, rmap, dcr) {
+  const status = r.status || "N/A";
```

Comment:
status is required so theoretically it'll never be null ... but N/A seems like a good fallback to me